### PR TITLE
JupyterLab intro suggestions

### PR DESCRIPTION
**Pull request by SimonKrughoff**

### CODE REVIEW

**Review by MelissaGraham — APPROVED** [review]

**Comment by MelissaGraham** [comment]
These are great updates Simon thank you so much!

File: data-access-analysis-tools/index-nb-intro.rst [hunk 1]
```diff
@@ -60,7 +60,8 @@ Pressing the orange start button to start the server returns this page with a bl
 **JupyterLab Navigation:**
 The JupyterLab landing page in the figure below is the launch pad for all JupyterLab functionality (e.g., Notebook, Terminal).
 Return to this launch pad at any time by clicking the plus symbol at upper left.
-For DP0.1, only Python Notebooks and the terminal interface are supported (i.e., not the Console), and users are not able to access Portal queries from a Jupyter Notebook.
+For DP0.1, most users will prefer using the Notebook or Terminal, but there is also a Python Console available.
+In order to duplicate catalog queries executed in the Portal, re-execute the query using the TAP client available in the Notebook
```
Comment (gpdf) [inline data-access-analysis-tools/index-nb-intro.rst:64]: We can do even better than this already.  I will try to provide a demo notebook by next week.  It's very simple; it's just that it's not at the top of my DP0.1 stack.

File: data-access-analysis-tools/index-nb-intro.rst [hunk 2]
```diff
@@ -76,12 +77,22 @@ All users will find a set of tutorial notebooks provided in the notebooks direct
 Jupyter Notebooks can be identified by their suffix ".ipynb". 
 
 **Using a Terminal:**
```
Comment (gpdf) [inline data-access-analysis-tools/index-nb-intro.rst:79]: I would suggest

**Using a JupyterLab Terminal:**

File: data-access-analysis-tools/index-nb-intro.rst [hunk 3]
```diff
@@ -76,12 +77,22 @@ All users will find a set of tutorial notebooks provided in the notebooks direct
...
-The DP0.1 data sets and the LSST Science Pipelines tools can both be accessed from the command line of a terminal. As described in the default message that appears in all newly-launched terminals, to create a Rubin Observatory environment in a terminal session and set up the full set of packages, users must first execute "source ${LOADSTACK}" and "setup lsst_distrib". For example, to query and retrieve DP0.1 data sets using the Butler (see below), command-line tools are available as documented `here <https://pipelines.lsst.io/v/weekly/modules/lsst.daf.butler/scripts/butler.html>`_. Type "butler --help" in any terminal to see a list of available butler functionality.
+The DP0.1 data sets and the LSST Science Pipelines tools can both be accessed from the command line of a terminal.
```
Comment (gpdf) [inline data-access-analysis-tools/index-nb-intro.rst:80]: "... of a JupyterLab Terminal tab."

File: data-access-analysis-tools/index-nb-intro.rst [hunk 4]
```diff
@@ -76,12 +77,22 @@ All users will find a set of tutorial notebooks provided in the notebooks direct
...
+
+   source ${LOADSTACK}
+   setup lsst_distrib 
+
+For example, to query and retrieve DP0.1 data sets using the Butler (see below), command-line tools are available as documented `here <https://pipelines.lsst.io/v/weekly/modules/lsst.daf.butler/scripts/butler.html>`_.
+Type ``butler --help`` in any terminal to see a list of available butler functionality.
 
 **Safely Log Out of JupyterLab:**
 Use the File item in the top menu bar.
 To safely shut down a Notebook, choose Close and Shutdown Notebook.
```
Comment (gpdf) [inline data-access-analysis-tools/index-nb-intro.rst:93]: We might want to give users a little advice on whether it's even possible to not save / to revert work that they've done in a notebook session.  I don't fully understand notebook auto-save behavior myself.

File: data-access-analysis-tools/index-nb-intro.rst [hunk 5]
```diff
@@ -95,7 +106,7 @@ The best way to learn how to use a Jupyter Notebook is to open the first of the
 Jupyter notebooks provide "cells" within which you type either Python code or markdown language (for formatted text).
 Choose the cell to execute by clicking in it with your mouse (the cursor must be in the desired cell).
 Hold down the `shift` key and press either `enter` or `return` (depending on your keyboard type), and the contents of the cell will be executed.
```
Comment (gpdf) [inline data-access-analysis-tools/index-nb-intro.rst:108]: "... type), or click the 'Play' button (▶️) in the notebook toolbar, and the ..."

**Review by gpdf — COMMENTED** [review]
Looks good, just a few suggestions (I know this has already been merged).

**Comment by MelissaGraham** [comment]
@gpdf implemented all suggestions Gregory thank you!

**Pull request merged**